### PR TITLE
aurora 1.2 (new formula)

### PR DESCRIPTION
**Pull request by xuri**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

aurorais a Beanstalkd queue server console. please help me to add this app to Homebrew, thanks a lot.

### CODE REVIEW

File: Formula/aurora.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Aurora < Formula
+  desc "Beanstalk queue server console"
+  homepage "https://xuri.me/aurora"
+  url "https://github.com/Luxurioust/aurora/releases/download/1.2/aurora_darwin_amd64_v1.2.tar.gz"
+  sha256 "c480e46592d6fb6f025572795bac80357f81b662cefdb3d149c77c723fb9a81f"
+
+  def install
+    bin.install "aurora"
```
Comment (zmwangx) [inline Formula/aurora.rb]: Need to build from source rather than download a precompiled binary.

**Review by zmwangx — CHANGES_REQUESTED** [review]

**Comment by zmwangx** [comment]
Also, repository is too new, need to wait until the 30 day threshold.

**Comment by MikeMcQuaid** [comment]
I'm fine with including it early given it's popular enough.

**Comment by dunn** [comment]
Here's how the test is failing:
```
==> aurora
To view beanstalk console open http://127.0.0.1:3000 in browser
Error: aurora: failed
execution expired

**Comment by dunn** [comment]
I'd try to update the test to ping the server with curl then kill it, like in [`algernon.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/algernon.rb)
.

**Comment by Luxurioust** [comment]
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
---

aurorais a Beanstalkd queue server console. please help me to add this app to Homebrew, thanks a lot.

**Pull request closed without merging**